src/realm/object-store/impl/realm_coordinator.cpp

```diff
@@ -913,13 +913,16 @@ namespace {
 bool compare_notifier_versions(const std::shared_ptr<_impl::CollectionNotifier>& a,
                                const std::shared_ptr<_impl::CollectionNotifier>& b)
 {
-    return a->version() < b->version();
+    if (a->version() < b->version()) {
+        return true;
+    }
+    return false;
 }
 
 class IncrementalChangeInfo {
 public:
     IncrementalChangeInfo(Transaction& sg, std::vector<std::shared_ptr<_impl::CollectionNotifier>>& notifiers)
-        : m_sg(sg)
+        : m_rt(sg)
```

Comment:
Wouldn't m_tr be better naming (tr for transaction?)

Reply:
(yes, I know m_rt can mean read-transaction, but...)

Reply:
I tossed a dice.